cocotb/_build_libs.py

```diff
@@ -126,6 +127,24 @@ def _get_python_lib():
 
     return python_lib
 
+
+# On Windows we generate import libraries that contains the code required to
+# load the DLL (.a) based on module definition files (.def)
+def _gen_import_libs(def_dir):
+
+    if os.name == "nt":
```

Comment:
I think this should be done by compiler name, perhaps - this seems like a mingw-specific thing. I'll look at numpy to try and work out how to do that.

Reply:
At the moment it will not work with anything else. I think if we add MSVS than one can think about this.

Reply:
Fair.

This should still go in the `build_ext` subclass though, otherwise it will run even if I'm not trying to build anything.